.github/workflows/debops-ci

```diff
@@ -0,0 +1,563 @@
+#!/usr/bin/env python3
+
+# Copyright 2020-2021 Federico Ceratto <federico@openobservatory.org>
+#                                      <federico@debian.org>
+# Released under GPLv3 license
```

Comment:
```suggestion
```

Let's keep the code under BSD 3-clause and the copyright to OONI. All the rest of the repository is released under these rules and there is no need that I can see to use different rules for this file specifically.